src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementFormView.tsx

```diff
@@ -213,15 +215,22 @@ const ReimbursementRequestFormView: React.FC<ReimbursementRequestFormViewProps>
               <input
                 onChange={(e) => {
                   if (e.target.files) {
-                    receiptAppend({
-                      file: e.target.files[0],
-                      name: e.target.files[0].name,
-                      googleFileId: ''
-                    });
+                    const file = e.target.files[0];
+                    if (file.size < 1000000) {
+                      receiptAppend({
+                        file: e.target.files[0],
+                        name: e.target.files[0].name,
+                        googleFileId: ''
+                      });
+                    } else {
+                      toast.error('File must be less than 1 MB', 5000);
+                      document.getElementById('receipt-image')!.value = '';
```

Comment:
huh theres no better way to do this? like using the form?

Reply:
because doesnt this wipe all the receipt images what if theres some images that arent over 1mb we only want to wipe the ones that are

Reply:
not that I could find, bc this is clearing the image stored in the input, not the value for 'receipts' that we're storing in the form. This just clears the imageData in the input, it doesn't remove any images we've already added as receipts